ert_shared/storage/extraction_api.py

```diff
@@ -157,7 +161,12 @@ def _dump_response(rdb_api, blob_api, responses, ensemble_name):
 
 def _extract_active_observations(facade):
     update_step = facade.get_update_step()
-    ministep = update_step[len(update_step) - 1]
+    ministep = None
+    try:
+        ministep = update_step[len(update_step) - 1]
```

Comment:
Does not `update_step` implement `__len__`?

Reply:
It does :of-course :)